### PR TITLE
Revert "remove alt text from screenshot (#469)"

### PR DESCRIPTION
**Pull request by JonLavi**
This reverts commit ec7ec7b52d4d3671e1d2129481c5b3250cb7f84b.

Restoring the alt text for screenshots in some of the readmes that previously removed as it was causing issues with the sample viewer.

### CODE REVIEW

File: map/download-preplanned-map/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Take a map offline using a preplanned map area.
 
-![](DownloadPreplannedMap.png)
+![Download Preplanned Map Sample](DownloadPreplannedMap.png)
```
Comment (Rachael-E) [inline map/download-preplanned-map/README.md]:
```suggestion
![Image of Download Preplanned Map](DownloadPreplannedMap.png)
```

File: map/honor-mobile-map-package-expiration-date/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 Determine a mobile map package's expiration date.
 
-![](HonorMobileMapPackageExpirationDate.png)
+![Honor mobile map package expiration date sample](HonorMobileMapPackageExpirationDate.png)
```
Comment (Rachael-E) [inline map/honor-mobile-map-package-expiration-date/README.md]:
```suggestion
![Image of honor mobile map package expiration date](HonorMobileMapPackageExpirationDate.png)
```

File: symbology/custom-dictionary-style/README.md [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 Use a custom dictionary style (.stylx) to symbolize features using a variety of attribute values.
 
-![](CustomDictionaryStyle.png)
+![Custom dictionary style](CustomDictionaryStyle.png)
```
Comment (Rachael-E) [inline symbology/custom-dictionary-style/README.md]:
```suggestion
![Image of custom dictionary style](CustomDictionaryStyle.png)
```

**Review by Rachael-E — APPROVED** [review]
thanks @JonLavi, just 3 to bring up to the "standard" description (including "image of" in the description) then good to merge, thanks!

**Comment by JonLavi** [comment]
Thanks @Rachael-E , I think those alt-texts you have spotted were still on the 'to do' list for the readmes, but now that we have them here it's already sorted!

**Review by tschie — APPROVED** [review]

**Pull request merged**